packages/cozy-client/src/withMutations.jsx

```diff
@@ -12,8 +12,8 @@ const makeMutationsObject = (mutations, client, props) => {
 
 /**
  * @function
- * @description HOC to provide mutations to components. Needs client in context
- * or as prop.
+ * @description HOC to provide mutations to components. Needs client in context or as prop.
+ * This is deprecated, prefer to use withClient and access directly the client.
```

Comment:
Can you add a @deprecated tag? https://jsdoc.app/tags-deprecated.html

Reply:
TIL, thanks